README.md

```diff
@@ -251,6 +251,10 @@ environment variable:
   can recognize false positives by "<untagged>" occurring in the message -- this
   indicates a pointer that was cast from an integer, so Miri was unable to track
   this pointer.
+* `-Zmiri-compare-exchange-weak-failure-rate=<rate>` changes the failure rate of
+  weak atomic operations. The default is `0.8` (so 4 out of 5 weak ops will fail.
```

Comment:
"weak atomic" is ambiguous, this could also refer to "weak memory", i.e., "everything except for `SeqCst`.
```suggestion
  `compare_exchange_weak` operations. The default is `0.8` (so 4 out of 5 weak ops will fail.
```